src/pages/hcp/api-docs/waypoint/[[...page]].tsx

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+// Lib
+import { fetchCloudApiVersionData } from 'lib/api-docs/fetch-cloud-api-version-data'
+// View
+import OpenApiDocsView from 'views/open-api-docs-view'
+import {
+	getStaticPaths,
+	getStaticProps as getOpenApiDocsStaticProps,
+} from 'views/open-api-docs-view/server'
+// Types
+import type { GetStaticProps, GetStaticPropsContext } from 'next'
+import type { OpenAPIV3 } from 'openapi-types'
+import type {
+	OpenApiDocsParams,
+	OpenApiDocsViewProps,
+	OpenApiDocsPageConfig,
+} from 'views/open-api-docs-view/types'
+import { schemaModShortenHcp } from 'views/open-api-docs-view/utils/massage-schema-utils'
+
+/**
+ * OpenApiDocsView server-side page configuration
+ */
+const PAGE_CONFIG: OpenApiDocsPageConfig = {
+	productSlug: 'hcp',
+	serviceProductSlug: 'waypoint',
+	basePath: '/hcp/api-docs/waypoint',
+	githubSourceDirectory: {
+		owner: 'hashicorp',
+		repo: 'hcp-specs',
+		path: 'specs/cloud-waypoint-service',
+		ref: 'main',
+	},
+	groupOperationsByPath: true,
+	statusIndicatorConfig: {
+		pageUrl: 'https://status.hashicorp.com',
+		endpointUrl:
+			'https://status.hashicorp.com/api/v2/components/1zyf32pwbfmz.json',
+	},
+	navResourceItems: [
+		{
+			title: 'Tutorial Library',
+			href: '/tutorials/library?product=waypoint',
+		},
```

Comment:
should we add Tutorial library here? There are no results for it ([link](https://dev-portal-git-zshcp-waypoint-api-docs-hashicorp.vercel.app/tutorials/library?product=waypoint))

Reply:
Ahh great catch, of course 🤦 I'll update tomorrow to link to https://developer.hashicorp.com/waypoint/tutorials/hcp-waypoint ... (or I guess we could remove entirely, I'm not sure!)

Reply:
I think for ui consistency, we _should_ have it. Maybe `Tutorials` instead of `Tutorial Library`? WDYT?

```suggestion
		{
			title: 'Tutorials',
			href: '/https://developer.hashicorp.com/waypoint/tutorials/hcp-waypoint
		},
```